tests/lib/external/snapd-testing-tools/tools/os.query

```diff
@@ -71,6 +72,22 @@ is_core24() {
     fi
 }
 
+is_core_gt() {
+    compare_ubuntu "${1:-}" "-gt"
```

Comment:
Using compare_ubuntu on ubuntu core will results in "comparing non ubuntu system", because grep -x looks for exact match of 'ID=ubuntu'.